README.md

```diff
@@ -32,7 +62,12 @@ await server.register({
 
 - `traceUUIDHeader`: defaults to 'x-gg-trace-uuid'. The header that is used for the Trace ID
 - `traceSeqIDHeader`: defaults to 'x-gg-trace-seqid' The header that is used for the Sequence ID
-
+- `cache`: internal memory cache settings. See [node-cache](https://github.com/node-cache/node-cache)
```

Comment:
> * Added in in memory cache that is used to pass the tracer information between the server, request and logger. This is set to have a TTL per object that is reset on each `get` of the object. If an object is stale for the length of the TTL, it will be culled. There is a max number of keys that can be active at any time to help with memory concerns.

Can you add something like this description to the readme?  You have nice descriptions of the cache configuration options but not what the cache is used for or why you would tweak these options.

Reply:
Updated